### PR TITLE
[AG-666]: Wombat simulator

### PR DESCRIPTION
**Pull request by guiltylotus**

### CODE REVIEW

File: pkg/source/wombat/pools_list_updater.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+			assets {
+			  id
+			  underlyingToken {
+				decimals
+				id
+			  }
+			}
+			createdTimestamp
+		  }
+	}`, lastCreateTime),
```
Comment (jack-wombat) [inline pkg/source/wombat/pools_list_updater.go:152]: This should filter out assets that are paused. i.e. `not isPaused`. (Note: isPaused can be undefined, true, or false.)

FYI, this is the query 1inch uses. To ensure correctness, please compare your query against this one after you add the filter. One thing to keep in mind is that our deprecated assets are moved under standalone pool for withdrawals -- there assets will be paused. (pool address = `0x6569DDC1Cc2648c89BC8025046A7dd65EB8940F3` on BSC; `0xD64816Fbdf50a1C4AEa456A4006ad21A928305f3` on ARB)
```
{
  assets(where: {isPaused_not: true}) {
  id
  poolAddress
  underlyingToken {
    id
  }
  liability
  }
}
```

File: pkg/source/maverickv1/pool_list_updater.go [hunk 2]
```diff
@@ -73,9 +73,9 @@ func (d *PoolListUpdater) getNewPoolFromSubgraph(ctx context.Context, lastCreate
 	subgraphPools, err := d.querySubgraph(ctx, lastCreateTime, d.config.NewPoolLimit, 0)
 	if err != nil {
 		logger.WithFields(logger.Fields{
-			"type":  poolABI,
+			"type":  DexTypeMaverickV1,
```
Comment (hanguyenk) [inline pkg/source/maverickv1/pool_list_updater.go:76]: `DexTypeMaverickV1` ?

Comment (guiltylotus) [reply pkg/source/maverickv1/pool_list_updater.go:76]: I saw the wrong logger info in Maverick folder :( So I updated it.

File: pkg/source/wombat/pools_list_updater.go (hunk 1 above)
Comment (guiltylotus) [reply pkg/source/wombat/pools_list_updater.go:155]: Thanks for you comment, I updated it.

File: pkg/source/wombat/math.go [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+		}
+	}
+
+	return new(big.Int).Mul(high, toWadFactor), nil
+}
+
+func poolV2QuoteFactor(fromAsset, toAsset Asset) *big.Int {
+	// virtual function; do nothing
+	return big.NewInt(1e18)
+}
```
[the remark this answers is not in the document]
Comment (guiltylotus) [reply pkg/source/wombat/math.go]: Updated. Thank you!

**Review by hanguyenk — APPROVED** [review]

**Pull request merged**